### PR TITLE
Add info about enabling plugin during manual installation

### PR DESCRIPTION
**Pull request by komik966**

### CODE REVIEW

**Comment by mickmister** [comment]
Thanks for the contribution @komik966!

However I don't think the step added in this PR is necessary to enable/disable a plugin. You should be able to click `Enable` or `Disable` in the system console to enable/disable individual plugins. Can you try this to make sure this works in your environment?

image

**Comment by komik966** [comment]
@mickmister Yes, this also works in my environment.
When I followed current documentation about manual plugin installation - it didn't worked. After debugging I realized that plugin can be enabled / disabled :)
So, I would consider adding this step but describe here that plugin needs to be enabled in system console (not through `config.json` as I described)

**Comment by hanzei** [comment]
@mickmister The documents mentions the starter template: 

> Note that the steps below are intentionally very manual to explain all of the pieces fitting together. In practice, we recommend referencing mattermost-plugin-starter-template for helpful build scripts.

2/5 that this phrase is enough and we should keep the document as it is.

**Review by mickmister — APPROVED** [review]
LGTM :+1: Thanks @komik966!

**Review by hanzei — APPROVED** [review]

File: site/content/extend/plugins/webapp/hello-world.md [hunk 1]
```diff
@@ -175,5 +175,6 @@ Install the plugin in one of the following ways:
                 main.js
     ```
     - Restart the Mattermost server.
+    - Enable the plugin in System Console UI
```
Comment (justinegeffen) [inline site/content/extend/plugins/webapp/hello-world.md:178]:
```suggestion
    - Enable the plugin in **System Console > Plugins (Beta) > Plugin Management**.
```

**Comment by justinegeffen** [comment]
Thanks for this, @komik966! I've provided a suggestion for the workflow. LMK if you have any questions!

**Comment by komik966** [comment]
@justinegeffen Good suggestion. I've updated.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**